main.go

```diff
@@ -0,0 +1,154 @@
+package main
+
+import (
+	"bufio"
+	"os"
+	"strings"
+	"strconv"
+	"net/http"
+    "github.com/gin-gonic/gin"
```

Comment:
Be careful with the indentation as it may not be relevant for the compiler, it is a good practice, if using visual code just download the go extension

Reply:
also separate with a white line as follow:

import (
    "core packages"

    "your packages"

    "third party packages"
)